src/App.tsx

```diff
@@ -1,19 +1,26 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.scss';
 import {Navigation} from './TempNav';
 import {BrowserRouter as Router, Route, Switch} from "react-router-dom";
 import {Curiosity} from "./Pages/CuriosityLandingPage/CuriosityLandingPage";
 import {Spirit} from "./Pages/SpiritLandingPage/SpiritLandingPage";
 import {Opportunity} from "./Pages/OpportunityLandingPage/OpportunityLandingPage";
-
+import {getImageOfTheDay} from "./Components/ApiFetch/ApodApiFetch";
 
 function App() {
+    const [image, setImage] = useState<string>("");
+    
+    useEffect(() => {
+        getImageOfTheDay().then(imageOfTheDay => setImage(imageOfTheDay.url));
+    }, []);
+    
   return (
       <Router>
             <div>
               <header className="header">
                 <nav>
                     <h1>Mars Mission</h1>
+                    {image}
```

Comment:
sorry - my fault but I'm not actually sure that this is the right place to put the image.
Shouldn't it be on the landing page only?